test/robot/testsuites/create_device.robot

```diff
@@ -47,7 +47,28 @@ Check That Update Entity Attributes Updates Values Correctly
     ${deviceValue}=     Get From Dictionary     ${resp.json()}     value
     ${value}=           Get From Dictionary     ${deviceValue}     value
 
-    Should Be Equal As Strings   snow=25;t=12  ${value}
+    Should Be Equal As Strings   snow%3D25%3Bt%3D12  ${value}
```

Comment:
Is there a helper for URL encoding that we can use? That could make the test a bit more readable.